tests/main.c

```diff
@@ -14,7 +14,7 @@
 int main(int argc, char **argv) {
     RET_TYPE ret;
     pcc_context_t *ctx = pcc_create(NULL);
-    while (pcc_parse(ctx, &ret));
+    while (pcc_parse(ctx, &ret)) {}
```

Comment:
I had to add empty block instead of semicolon. Otherwise the formatter kept pushing the semicolon to the next line and indenting it. But I think it is better for readability this way. It emphasizes that the loop body is empty.